docs/pages/includes/plugins/identity-export.mdx

```diff
@@ -1,23 +1,22 @@
-Like all Teleport users, `access-plugin` needs signed credentials in
+Like all Teleport users, `{{ user }}` needs signed credentials in
```

Comment:
Is this some new templating language, or will this literally show `{{ user }}` in the docs?

Reply:
It's how we add variables when calling partials. See the [add-role-to-user](https://github.com/gravitational/teleport/blob/master/docs/pages/includes/add-role-to-user.mdx) partial for an existing example.